### ISSUE TITLE
Fix loss spikes

### ISSUE THREAD

**Issue by danbraunai** [goodfire-ai/spd#87]
In the run on all MLP layers of SS 1.25M ([here](https://wandb.ai/goodfire/spd/reports/Simple-Stories-1-25M-MLPs--VmlldzoxMzc2MjAyMQ)), we're seeing a lot of loss spikes. We should see if reducing the learning rate fixes this. Otherwise we might want to look into initilization (I think @oli-clive-griffin had some unmerged tweaks here) and/or optimizer strategies that are known to stabilize Adam during training.

**Comment by danbraunai** [comment]
A smaller learning rate at least reduces the spikes https://wandb.ai/goodfire/spd/runs/37lwt4fr. We can possibly remove them entirely with an even smaller learning rate.

Though note that it would be desirable if we could come up with another fix that allows for higher learning rates. This would be important but lower priority if we do manage to fix the loss spikes with a smaller lr.
